### PR TITLE
Update union-value

### PR DESCRIPTION
**Pull request by johlym**
Fixes Advisory 1012 via `set-value`

https://nodesecurity.io/advisories/1012

### CODE REVIEW

**Comment by johlym** [comment]
This is absolutely likely to break `group-array` for Node.JS 5 and older. Those versions haven't been supported for a while. I'd be happy to denote that in the readme, as well.

**Comment by doowb** [comment]
Thanks, I'll take a look at this tonight.

**Comment by doowb** [comment]
@johlym will you try `union-value@1.0.1`. This is a patched version that uses the patched `set-value@2.0.1` version. Those are both compatible with older versions of Node.js.

Once that's working and I can release a patch of `group-array`, then I can update the versions to the latest and publish a major release of `group-array`.

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "get-value": "^2.0.6",
     "kind-of": "^3.1.0",
     "split-string": "^1.0.1",
-    "union-value": "^0.2.3"
+    "union-value": "^2.0.1"
```
Comment (doowb) [inline package.json]:
```suggestion
    "union-value": "^1.0.1"
```

**Comment by johlym** [comment]
Yeah this looks good on my end.

**Comment by doowb** [comment]
Thanks! I'll merge this now and get it published later today.

**Comment by doowb** [comment]
@johlym I published `group-array@0.3.4`. If you'd like to do a PR to get `group-array` updated to latest versions of the dependencies, that would be awesome :) We'd be able to publish it as a major then.

**Pull request merged**